reference-adapter/pspcl-ifix-adapter/src/main/resources/application.properties

```diff
@@ -0,0 +1,43 @@
+app.timezone=UTC
+
+#----------------Postgres Configurations----------------#
+spring.datasource.driver-class-name=org.postgresql.Driver
+spring.datasource.url=jdbc:postgresql://localhost:5432/pspcl_ifix
+spring.datasource.username=postgres
+spring.datasource.password=root
+#spring.jpa.hibernate.ddl-auto=update
+spring.jpa.properties.hibernate.jdbc.lob.non_contextual_creation= true
+spring.jpa.properties.hibernate.dialect= org.hibernate.dialect.PostgreSQLDialect
+
+#----------------Uri Configurations----------------#
+pspcl.fetch.bill.url=https://billapp.pspcl.in//fetchbillingdata.asmx
+pspcl.fetch.payment.url=https://billapp.pspcl.in//fetchbillingdata.asmx
+
+#----------------ifix core url config----------------#
+ifix.host=https://ifix-qa.ifix.org.in
+ifix.event.url=/fiscal-event-service/events/v1/_publish
+
+#----------------QA-KeyCloak credentials----------------#
+keycloak.host=https://ifix-qa.ifix.org.in
+keycloak.token.url=/auth/realms/ifix/protocol/openid-connect/token
+keycloak.credentials.clientid=pspcl-ifix-qa
+keycloak.credentials.clientsecret=7e10ca5f-e0d2-47cc-b40e-ddf244bea211
+keycloak.credentials.granttype=client_credentials
+
+fiscal.event.tenantId=pb
+demand.coaCode=8247-12-373-56-78-90
+receipt.coaCode=8247-12-373-56-78-90
+fiscal.event.receiver=mGramSeva
```

Comment:
This will be the client-id like "mgramseva-qa". The client-id is the username registered in keycloak.

Reply:
done. will push once all the comments are resolved.